webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/list/ActivityStreamList.vue

```diff
@@ -126,6 +127,7 @@ export default {
   created() {
     document.addEventListener('activity-deleted', event => {
       const activityId = event && event.detail;
+      this.isDeleted = true;
```

Comment:
```suggestion
  if (this.isActivityDetail) {
      this.isDeleted = true;
  }
```
If you have one single activity inside the stream of a space and then delete it, we should display `activity-stream-empty-message-space` and not an empty content. consequently, we need to change this variable only when we are in a context of single activity display.

Reply:
done!!